src/frontend/hooks/useSelectFileAndImportConfig.ts

```diff
@@ -24,7 +27,12 @@ export function useSelectFileAndImportConfig() {
       }
     },
     onSuccess: () => {
-      return queryClient.invalidateQueries({queryKey: [PROJECT_SETTINGS_KEY]});
+      return Promise.all([
+        queryClient.invalidateQueries({queryKey: [FIELDS_KEY]}),
+        queryClient.invalidateQueries({queryKey: [ICONS_KEY]}),
+        queryClient.invalidateQueries({queryKey: [PROJECT_SETTINGS_KEY]}),
+        queryClient.invalidateQueries({queryKey: [PRESETS_KEY]}),
+      ]);
```

Comment:
I _think_ it makes sense to have the mutation wait for all of these to resolve. the tradeoff is a slightly longer mutation time in the `active` state when importing a config